### PR TITLE
fix alpine base update docs

### PR DESCRIPTION
**Pull request by deitch**
Signed-off-by: Avi Deitcher <avi@deitcher.net>

**- What I did**

Fixed a few misleading lines on the alpine base update doc

**- How I did it**

markdown

**- How to verify it**

Read it

**- Description for the changelog**

update docs

### CODE REVIEW

File: docs/alpine-base-update.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ following which is an explanation of each one.
 ```sh
 # Update tools packages
 cd $LK_ROOT/tools
-$LK_ROOT/scripts/update-component-sha.sh --pkg $LK_ROOT/tools/alpine
+$LK_ROOT/scripts/update-component-sha.sh --image $LK_ALPINE
```
Comment (rn) [inline docs/alpine-base-update.md:133]: what's the reason for this change? Does `update-component-sha.sh` no longer take the `--pkg` option?

**Review by rn — APPROVED** [review]

File: docs/alpine-base-update.md (hunk 1 above)
Comment (deitch) [reply docs/alpine-base-update.md:133]: It does, but `tools/alpine` does not have a `build.yml`. To be fair, I have no idea how it worked before, but if this works, and we already are storing the `LK_ALPINE` var, might as well use it.
